src/builder/edit_channel.rs

```diff
@@ -72,4 +73,21 @@ impl EditChannel {
 
         self
     }
+
+    /// The parent category of the channel.
+    ///
+    /// This is for [text] and [voice] channels only.
+    ///
+    /// [text]: ../model/enum.ChannelType.html#variant.Text
+    /// [voice]: ../model/enum.ChannelType.html#variant.Voice
+    pub fn category<C>(mut self, category: C) -> Self
+        where C: Into<Option<ChannelId>> {
```

Comment:
This should be just `ChannelId`, as since this is a builder the `Option` part becomes more or less implicit.

Reply:
How would you take a channel out of a category, then? As far as I understand it, since the PATCH method has all parameters optional, leaving the `parent_id` field out would not change anything.

Reply:
Oh, misunderstood the reasoning for the option 
(Since it seemed to almost do the same thing as serde, but explicitly). 
i'll merge it as is then.